elytron-oidc-client/subsystem/src/main/resources/org/wildfly/extension/elytron/oidc/LocalDescriptions.properties

```diff
@@ -1,144 +1,142 @@
+elytron-oidc-client.secure-deployment.ignore-oauth-query-parameter=Disable query parameter parsing for the 'access_token'. Users will not be able to authenticate if they only pass in an 'access_token'.
 elytron-oidc-client.secure-deployment.proxy-url=The URL for the HTTP proxy if one is used.
-elytron-oidc-client.secure-deployment.verify-token-audience=If true, then during bearer-only authentication, the adapter will verify if token contains this client name (resource) as an audience
-elytron-oidc-client.secure-deployment.adapter-state-cookie-path=If set, defines the path used in cookies set by the adapter. Useful when deploying the application in the root context path.
-elytron-oidc-client.secure-deployment.credential=Credential used to communicate with the OpenID Connect provider
-elytron-oidc-client.secure-deployment.credential.add=Add credentials used to communicate with the OpenID Connect provider
-elytron-oidc-client.secure-deployment.credential.remove=Remove credentials used to communicate with the OpenID Connect provider
-elytron-oidc-client.secure-deployment.credential.client-key-password=Password for the client key. This is required if client-keystore-file has been specified.
-elytron-oidc-client.secure-deployment.credential.client-keystore-password=Password for the client keystore. This is required if client-keystore-file has been specified.
-elytron-oidc-client.secure-deployment.credential.client-keystore-file=Path to the client keystore. This is optional.
-elytron-oidc-client.secure-deployment.credential.client-keystore-type=Client keystore type
-elytron-oidc-client.secure-deployment.credential.client-key-alias=Client key alias
-elytron-oidc-client.secure-deployment.credential.token-timeout=Token timeout
-elytron-oidc-client.secure-deployment.credential.secret=Secret
-elytron-oidc-client.secure-deployment.credential.algorithm=Algorithm for signed JWT
-elytron-oidc-client.secure-deployment.redirect-rewrite-rule.add=Add a rewrite rule for the redirect URI
-elytron-oidc-client.secure-deployment.redirect-rewrite-rule.remove=Remove a rewrite rule for the redirect URI
-elytron-oidc-client.secure-deployment.redirect-rewrite-rule.replacement=The replacement string
-elytron-oidc-client.secure-deployment.socket-timeout-millis=Timeout for socket waiting for data in milliseconds
-elytron-oidc-client.secure-deployment.connection-ttl-millis=Connection time to live in milliseconds
-elytron-oidc-client.secure-deployment.connection-timeout-millis=Timeout for establishing the connection with the remote host in milliseconds
-elytron-oidc-client.secure-deployment.token-signature-algorithm=The token signature algorithm used by the OpenID provider
-
-
+elytron-oidc-client.secure-deployment.verify-token-audience=If set to 'true', then during bearer-only authentication, verifies if the token contains the client name defined as an audience. It is recommended to set to 'true' for improved security.
```

Comment:
s / , verifies / , the subsystem verifies
s /  set to 'true' / set the value to 'true'